cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -0,0 +1,5267 @@
+[
+  {
+    "abstract": {
+      "description": "Index"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
```

Comment:
Record 451 points to the CMS collaboration author list, see https://opendata.cern.ch/record/451

I guess you meant to point to 26050 which is the record ID for JADE author list for preserved data: https://opendata-qa.cern.ch/record/26050

Note though that this is not a complete list of computing notes, perhaps we should have an "all-hands'" list of all JADE people?

Reply:
Ok, then this would need correction in the loogbook file as well.